modules/ensembl/annotation/dumpanno/meta.yml

```diff
@@ -0,0 +1,51 @@
+---
+name: "annotation_dumpanno"
+
+description: This module is used to dump the functional annotation of a genome from core databases.
+
+keywords:
+  - container
+  - core_database
+  - docker
+  - ensembl-legacy-scripts
+  - fasta
+  
+tools:
+  - "annotation":
+      homepage: "https://github.com/Ensembl/nextflow_modules"
+      description: "This module handles all annotation-related interactions with the core database, including annotation dumping and processing"
+      license: "Apache License version 2.0"
```

Comment:
```suggestion
      license: ['Apache License version 2.0']
```

Reply:
So sorry Disha, I made a mistake changing this from how you had it. I will have to go back to my own PR and revert this as well. So just to be in sync.